src/main/java/org/broadinstitute/hellbender/tools/exome/CreatePanelOfNormals.java

```diff
@@ -187,103 +182,95 @@
     )
     protected File outFile = null;
 
+    @Argument(
+            doc = "Output file for sample names that failed the QC check (one sample name per line; sample names drawn from input file).  This option is ignored if QC step is being skipped.  Final PoN will not include these samples, if QC is enabled.",
+            shortName = BLACKLIST_QC_SHORT_NAME,
+            fullName = BLACKLIST_QC_FULL_NAME,
+            optional = true
+    )
+    protected File blacklistOutFile = null;
+
+    @Argument(
+            doc = "Output file for the weights of each target.  This is the 1/var(post-projected targets for each normal)." +
```

Comment:
Personally, I would rather store the weights in the PoN, even at the cost of forcing our command line tool generating a weights file to pass to CBS on the fly, than have an extra file floating around.  I assume the performance hit would be negligible compared to the runtime of CBS, right?